src/interfaces/business-lookup-interfaces.ts

```diff
@@ -1,4 +1,4 @@
-import { CorpTypeCd, EntityStates } from '@/enums'
+import { CorpTypeCd, EntityStates } from '@bcrs-shared-components/enums'
```

Comment:
`npx link` had issues with @/enum it wouldn't resolve easily, this syntax was used in another file.